integration_tests/CMakeLists.txt

```
@@ -2412,6 +2412,7 @@ RUN(NAME common_20 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME common_21 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME common_22 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
 RUN(NAME common_23 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
+RUN(NAME common_substring_01 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
```

Note: this should just use a numerical name, to make the tests uniform, so:

```suggestion
RUN(NAME common_24 LABELS gfortran llvm llvm_wasm llvm_wasm_emcc)
```